lib/utils.pm

```diff
@@ -304,7 +304,7 @@ sub fully_patch_system {
 sub minimal_patch_system {
     my (%args) = @_;
     $args{version_variable} //= 'VERSION';
-    if (sle_version_at_least('12-SP1', version_variable => $args{version_variable})) {
+    if (sle_version_at_least('12-SP1', version_variable => $args{version_variable}) && !(check_var('VERSION', 'SLES-11'))) {
```

Comment:
Is it possible to extend sle_version_at_least function to support sle_version_at_least('12')? If so, no change is required here.